### PR TITLE
chore: Display Java Diff between Monorepo and Split Repos

### PR DESCRIPTION
**Pull request by lqiu96**
WIP

### CODE REVIEW

File: java-compute/google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/BaseTest.java [hunk 1]
```diff
@@ -22,9 +22,8 @@ public class BaseTest {
   protected static final String DEFAULT_PROJECT = ServiceOptions.getDefaultProjectId();
   protected static final String DEFAULT_ZONE = "us-central1-a";
   protected static final String DEFAULT_REGION = "us-west1";
-  protected static final String COMPUTE_PREFIX = "it-test-compute";
 
   public static String generateRandomName(String placeholder) {
-    return COMPUTE_PREFIX + "-" + placeholder + "-" + UUID.randomUUID().toString().substring(0, 8);
+    return "gapic-" + placeholder + UUID.randomUUID().toString().substring(0, 8);
```
Comment (suztomo) [inline java-compute/google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/BaseTest.java:27]: Manual commit to monorepo https://github.com/googleapis/google-cloud-java/commit/8e4378ddb38c2ff4d50b65962ce729c2b237c095

File: java-container/google-cloud-container/src/test/java/com/google/cloud/container/v1/it/ITSystemTest.java [hunk 2]
```diff
@@ -26,7 +26,6 @@
 import com.google.container.v1.ListOperationsResponse;
 import com.google.container.v1.NodePool;
 import com.google.container.v1.Operation;
-import com.google.container.v1.Operation.Status;
```
Comment (suztomo) [inline java-container/google-cloud-container/src/test/java/com/google/cloud/container/v1/it/ITSystemTest.java:29]: Manual commit to monorepo. OK to have diff https://github.com/googleapis/google-cloud-java/commit/8e4378ddb38c2ff4d50b65962ce729c2b237c095

**Comment by suztomo** [comment]
You want to have the pull request on the diff script and Java code diff separated.

**Comment by lqiu96** [comment]
### Data-Catalog Diff
Split Repos: Changes came from https://github.com/googleapis/java-datacatalog/pull/795
Monorepo: Changes were reverted in https://github.com/googleapis/google-cloud-java/pull/8340

Create a PR to revert changes that have been reverted: https://github.com/googleapis/google-cloud-java/pull/8467

**Comment by suztomo** [comment]
The inifinite diff probably is caused by the recent protobuf changes applied to split repos.

**Comment by lqiu96** [comment]
Java Diffs (10/10/21):
### java-bigquerydatapolicy
This is a mystery. There modified files do not exist in the split repos

### java-dataplex
Owl-Bot PR in Monorepo (Currently Do Not Merge): https://github.com/googleapis/google-cloud-java/pull/8494

### java-distributedcloudedge
PR in split repos (not merged yet): https://github.com/googleapis/java-distributedcloudedge/pull/7

### java-gke-connect-gateway
Currently a mystery

### java-publicca
There is not corresponding split-repo. This diff is expected.

**Pull request closed without merging**